src/imitation/algorithms/preference_comparisons.py

```diff
@@ -118,7 +118,7 @@ def __init__(
     def sample(self, steps: int) -> Sequence[TrajectoryWithRew]:
         # make a copy before shuffling
         trajectories = list(self._trajectories)
-        self.rng.shuffle(trajectories)
+        self.rng.shuffle(trajectories)  # type: ignore[arg-type]
```

Comment:
From https://github.com/HumanCompatibleAI/imitation/pull/594/files
```suggestion
        # numpy's typing thinks it can't accept a list here, but it can
        self.rng.shuffle(trajectories)  # type: ignore[arg-type]
```